mappings/net/minecraft/util/math/MatrixStack.mapping

```diff
@@ -0,0 +1,18 @@
+CLASS net/minecraft/class_4587 net/minecraft/util/math/MatrixStack
```

Comment:
Duplicate class, remove this one or the other 4f stack

Reply:
Fixed.